i18n/en.json

```diff
@@ -6,20 +6,20 @@
-	"wikibasereconcileedit-editendpoint-invalid-reconcile-parameter": "Invalid reconcile parameter. Expecting an array",
-	"wikibasereconcileedit-editendpoint-unsupported-reconcile-version": "The only supported reconciliation {{PLURAL:$2|version is|versions are}} $1",
-	"wikibasereconcileedit-editendpoint-invalid-reconcile-propertyid": "$1 requires a single $2 key mapped to a property ID, such as P123",
-	"wikibasereconcileedit-editendpoint-property-not-found": "Could not find property with label '$1'",
-	"wikibasereconcileedit-editendpoint-invalid-property-type-must-be-url": "$1 property $2 must be of type url. The datatype of the provided property is $3",
-	"wikibasereconcileedit-editendpoint-unspecified-entity-input-version": "Entity input version must be specified",
-	"wikibasereconcileedit-editendpoint-invalid-entity-input-version": "Unknown entity input version $1; the only supported {{PLURAL:$3|version is|versions are}} $2",
-	"wikibasereconcileedit-editendpoint-qualifiers-references-not-supported": "Qualifiers and References are not currently supported",
-	"wikibasereconcileedit-editendpoint-reconciliation-property-missing-in-statements": "Entity must have at least one statement for the reconciliation Property",
-	"wikibasereconcileedit-editendpoint-invalid-reconciliation-statement-type": "Reconciliation statement must contain a value",
-	"wikibasereconcileedit-editendpoint-invalid-request-body": "Invalid or empty JSON request body",
-	"wikibasereconcileedit-editendpoint-invalid-content-type": "Content-type should be $1",
-	"wikibasereconcileedit-reconciliationservice-matched-multiple-items": "The reconciliation is ambiguous; matched {{PLURAL:$2|item|$2 items}}: $1",
-	"wikibasereconcileedit-minimaliteminput-required-keys": "Statements missing '$1' key",
-	"wikibasereconcileedit-fullwikibaseiteminput-unsupported-type": "The only supported entity {{PLURAL:$2|type is|types are}} $1.",
+	"wikibasereconcileedit-invalid-content-type": "Content-type should be $1",
+	"wikibasereconcileedit-invalid-request-body": "Invalid or empty JSON request body",
+	"wikibasereconcileedit-invalid-entity-input-version": "Unknown entity input version $1; the only supported {{PLURAL:$3|version is|versions are}} $2",
+	"wikibasereconcileedit-invalid-property-type-must-be-url": "$1 property $2 must be of type url. The datatype of the provided property is $3",
+	"wikibasereconcileedit-invalid-reconcile-parameter": "Invalid reconcile parameter. Expecting an array",
+	"wikibasereconcileedit-invalid-reconcile-propertyid": "$1 requires a single $2 key mapped to a property ID, such as P123",
+	"wikibasereconcileedit-unspecified-entity-input-version": "Entity input version must be specified",
+	"wikibasereconcileedit-unsupported-reconcile-version": "The only supported reconciliation {{PLURAL:$2|version is|versions are}} $1",
+	"wikibasereconcileedit-unsupported-type": "The only supported entity {{PLURAL:$2|type is|types are}} $1.",
```

Comment:
“unsupported type” in the message key sounds overly generic to me – how about unsupported-_entity_-type?

Reply:
addressed